### ISSUE TITLE
Improve mobile Accessibility during elections

### ISSUE THREAD

**Issue by MSoeb** [OpenSlides/openslides-client#5524]
**Current behavior**
In the current state it is quite though for people with a view disability to recognize the correct line and election option, if an election is done with multiple candidates. Candidate and voting option is not alinged. Therefor user have to guess, if they are in the correct position. 

**Wanted behavior**
A solution should be found to improve the accessibility for people with a view disability.

**Additional context**
I will add a more useful example.

**Comment by Elblinator** [comment]
closed due to #4903
